libflux/flux-core/src/semantic/mod.rs

```diff
@@ -51,9 +51,64 @@ pub enum Error {
     Inference(#[from] nodes::Error),
 }
 
+/// An environment of values that are available outside of a package
+#[derive(Default, Debug, Clone, PartialEq)]
+pub struct ExportEnvironment {
```

Comment:
I wonder if this should be called `ImportEnvironment`?

Reply:
Maybe, this is not a great name, though I also use it to "export" the public API after a semantic graph has been analyzed.